src/foam/nanos/crunch/ServerCrunchService.java

```diff
@@ -56,18 +56,18 @@ public void start() {
 
     var prerequisiteCapabilityJunctionDAO =
       ((DAO) getX().get("prerequisiteCapabilityJunctionDAO"));
-    
+
     prerequisiteCapabilityJunctionDAO.listen(new Sink() {
       public void put(Object obj, Detachable sub) {
         // ???: could have a sequence id per capability to mimimize how
         //      much cache is invalidated
-        cacheSequenceId_.getAndIncrement();
+ //       cacheSequenceId_.getAndIncrement();
```

Comment:
Can you put these three back and get updated performance results? I hope commenting these out doesn't change much.

Reply:
I uncommented the 3 lines and the performance is not negatively affected